docs/_includes/ww-admin-tasks-content.md

```diff
@@ -26,7 +26,29 @@ Go to the ‘Roles’ tab in the admin menu.
 
 Here you can create a group of permissions, called a ‘role’. You should choose role titles and definitions that make sense to your own organisation.
 
-Once you have created a role, you can then assign a user to one or more roles to give that user access to all the permissions of those roles, without having to assign permissions individually for each user.
+Roles enable you to more easily assign permissions to different staff users. For example, you can give all customer services staff the same permsissions by assigning them the same role, without having to assign permissions individually for each user.
+
+Once you've created a role, use the tick boxes to assign its permissions.
+
+WasteWorks permsissions include:
+
+| Permission      | Description |
```

Comment:
The table works, but has no styling; it looks like we might have to put class='table' and then do the whole table as HTML, bit annoying.